### ISSUE TITLE
[Design] Update Accessibility Chart to show colors that can be used with a black background.

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1274]
**User Story:**
As Spark we want to update our Accessibility Chart so that we are showing colors that can be used with a black background.

**Notes:**
Requestor: SME: RV, Woods Support Room

**Current State:**
Accessibility Chart:
https://sparkdesignsystem.com/using-spark/foundations/color-accessibility
- according to that chart, the only color that can be used with a black background is orbit lightest. 

![image](https://user-images.githubusercontent.com/46761151/94449423-0e26e500-017a-11eb-90ef-b83eef29d46f.png)

**AC:**
- Image resolution will match for High Res & Page Image.
- Charts will be consistent with High Res & Page Image.
- Text on High Res & Page Image will be consistent 
- connect with PO to get Issue made for Engineering
